src/components/Welcome.jsx

```diff
@@ -1,16 +1,28 @@
 import React, { Component } from 'react';
 import { Link } from "react-router-dom";
+import queryString from 'query-string';
 
 class Welcome extends Component {
+
+    componentDidMount() {
+        const query = queryString.parse(location.search);
+        const data = JSON.parse(query.info || '{}');
+        localStorage.setItem('token', data.token);        
```

Comment:
@jo-rdan  good progress so far, but I would like if you don't mind instead of saving user token as token on local storage why not considering saving it as user because am saving user token as user or if you feel naming them as token is a better idea we can stick to it too. let me know what you think.

Reply:
@Jajabenit250 thank you for your feedback, however, I think naming it token can be best because we need to specify what it is. what do you think?

Reply:
@jo-rdan okay let user token I think it is better.